main.js

```diff
@@ -36,20 +36,16 @@ buttonSection.addEventListener('click', function(event) {
 
 savedPalettesSection.addEventListener('click', async function(event) {
     if(event.target.classList.contains('delete-button')) {
-        deleteModal.classList.toggle('hidden');
-        main.classList.toggle('block');
-        savedContainer.classList.toggle('block');
+        modalClassToggler()
         await getPromiseFromEvent(deleteModal, 'click')
         if(shouldDelete) {
             var eventTargetParent = event.target.parentNode
             var thisSavedPaletteIndex = Array.from(eventTargetParent.parentNode.children).indexOf(eventTargetParent);
             deletePalette(eventTargetParent, thisSavedPaletteIndex);
         } 
-        deleteModal.classList.toggle('hidden');
-        main.classList.toggle('block');
-        savedContainer.classList.toggle('block');
+        modalClassToggler()
         shouldDelete = false;
-    } else if (event.target.classList.contains('mini-box')) {
+    } else if(event.target.classList.contains('mini-box')) {
```

Comment:
The space between the if and the parentheses was removed. We should be consistent through the code to include or remove the space. I've been adding it because of the style guide: https://github.com/turingschool-examples/javascript/tree/main/es5#whitespace 
Here is a screenshot:
![image](https://user-images.githubusercontent.com/123283073/232248649-c17744dc-331a-480e-b09b-574bcebaa249.png)

Reply:
Yep!! Switching it back